stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/MediaAttachmentContent.kt

```diff
@@ -372,13 +376,15 @@ internal fun MediaAttachmentContentItem(
                 interactionSource = MutableInteractionSource(),
                 indication = rememberRipple(),
                 onClick = {
```

Comment:
having a click listener, which only works if the sync status is correct makes it so that we can click, have the click ripple, but not do anything. 

I would either go for: we disable the click entirely if the attachment is in the wrong state (e.g. failed upload or uploading) or we'd open the menu (as if we long clicked) that lets the user retry the upload.